app/models/specialist_document.rb

```diff
@@ -86,7 +86,11 @@ def update(params)
   def published?
     editions.any?(&:published?)
   end
-  alias_method :has_ever_been_published?, :published?
+
+  def has_ever_been_published?
+    return false if @editions.size == 1 && needs_exporting?
+    published?
```

Comment:
To close the loop, once we write the final draft, then we send the publish message to publishing-api and then we set the exported_at timestamp so only at the end of the publishing process will the document say it is both `published?` and no longer `needs_exporting?`